containers/configs/settings_local.ocp.py

```diff
@@ -0,0 +1,130 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: Copyright contributors to the OpenScanHub project.
+
+"""
+Instance-specific settings.
+
+CI instance
+"""
+
+import os
+
+DEBUG = True
+TEMPLATE_DEBUG = DEBUG
+
+ADMINS = (
+    ('Siteshwar Vashisht', 'svashisht@redhat.com'),
+)
+
+MANAGERS = ADMINS
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'openscanhub',
+        'USER': 'openscanhub',
```

Comment:
@nirik How can we put username, password and hostname for central staging and production databases here?

Reply:
You should be able to use a ansible template and set variables for those things, which get filled in from ansible's private repo when termplated.
ie, like https://pagure.io/fedora-infra/ansible/blob/main/f/roles/openshift-apps/elections/templates/fedora-elections.cfg#_13